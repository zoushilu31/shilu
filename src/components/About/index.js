import React from 'react';
import coverImage from "../../assets/cover/cover-image.png"


function About () {
    return (
        <section>
            <h1 id="about">
                About Me
            </h1>
            <img src={coverImage} className="my-2" style={{ width: "100%" }} alt="cover" />
        </section>

    );
    
}

export default About;
