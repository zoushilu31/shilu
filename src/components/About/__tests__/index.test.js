import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import About from '..';



afterEach(cleanup);
describe('About component', () => {
    //renders About test
    it('renders', () => {
        render(<About />);
    }
    );
    //first test
    //second test
    
it('mathes snapshot DOM nod structure', () => {

})
  })
  const { asFragment } = render(<About />);
  expect(asFragment()).toMatchSnapshot();
