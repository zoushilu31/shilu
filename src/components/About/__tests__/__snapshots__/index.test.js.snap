// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<DocumentFragment>
  <section>
    <h1
      id="about"
    >
      About Me
    </h1>
    <img
      alt="cover"
      class="my-2"
      src="cover-image.png"
      style="width: 100%;"
    />
  </section>
</DocumentFragment>
`;
